includes/class-amp-http.php

```diff
@@ -205,6 +205,11 @@ public static function get_amp_cache_hosts() {
 			if ( function_exists( 'idn_to_utf8' ) ) {
 				if ( version_compare( PHP_VERSION, '5.4', '>=' ) && defined( 'INTL_IDNA_VARIANT_UTS46' ) ) {
 					$domain = idn_to_utf8( $domain, IDNA_DEFAULT, INTL_IDNA_VARIANT_UTS46 ); // phpcs:ignore PHPCompatibility.FunctionUse.NewFunctionParameters.idn_to_utf8_variantFound, PHPCompatibility.Constants.NewConstants.intl_idna_variant_uts46Found
+				} elseif ( version_compare( PHP_VERSION, '7.2', '==' ) || version_compare( PHP_VERSION, '7.3', '==' ) ) {
```

Comment:
If only PHP 7.2 and 7.3 are to be targeted then this also needs to account for patch versions, yes? This would fail on PHP 7.2.1, right?

```suggestion
				} elseif ( version_compare( PHP_VERSION, '7.2', '>=' ) && version_compare( PHP_VERSION, '7.4', '<' ) ) {
```